sparse/src/KokkosSparse_BsrMatrix.hpp

```diff
@@ -1108,6 +1108,10 @@ template <typename... P>
 struct is_bsr_matrix<BsrMatrix<P...>> : public std::true_type {};
 template <typename... P>
 struct is_bsr_matrix<const BsrMatrix<P...>> : public std::true_type {};
+
+/// \brief Equivalent to is_bsr_matrix<T>::value.
+template <typename T>
+inline constexpr bool is_bsr_matrix_v = is_bsr_matrix<T>::value;
```

Comment:
Nice, this aligns well with that c++ does in the standard library, makes it handy and easy to remember